builtin/providers/aws/resource_aws_opsworks_instance.go

```diff
@@ -565,6 +565,10 @@ func resourceAwsOpsworksInstanceRead(d *schema.ResourceData, meta interface{}) e
 	for _, v := range instance.LayerIds {
 		layerIds = append(layerIds, *v)
 	}
+	layerIds, err = sortListBasedonTFFile(layerIds, d, "layer_ids")
```

Comment:
@mbfrahry will this cause any issues for people who are using the resource right now?